src/linear-memory.js

```diff
@@ -0,0 +1,36 @@
+import { HeapVerifier } from "./heap-verifier";
+
+export class LinearMemory {
+  constructor({ initial = 2048, maximum = 2048 }) {
+    //https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory
+    //shared is problematic if pointer moved (e.g. pointer not handle)
+    //this.memory = new WebAssembly.Memory({ initial, maximum, shared: true })
```

Comment:
I suggest deleting these comments, this was before I learned how to renew pointers.

Reply:
removed